meteor-app/imports/ui/assessment2020/serviceItem-tag.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+// import { action } from '@storybook/addon-actions'
```

Comment:
We should wire up actions to the tags, so we can see they are working. DIscussion may be required to explain

Reply:
This one I may have to do it later because Murtaza is still working on tags.